src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -89,41 +84,44 @@ export class Conversations extends React.Component<Props, State> {
     if (!me) {
       return []
     }
+
     const conversations = me.conversations.edges
       .filter(({ node }) => {
         return node && node.last_message
       })
       .map(edge => edge.node)
+
     return conversations || []
   }
 
   fetchData() {
-    if (this.state.fetchingNextPage) {
+    if (this.props.relay.isLoading()) {
       return
     }
-    this.setState({ fetchingNextPage: true })
+
     this.props.relay.loadMore(PageSize, error => {
       // TODO: Not performing any error handling here
       this.setState({
-        fetchingNextPage: false,
         dataSource: this.state.dataSource.cloneWithRows(this.conversations),
       })
     })
   }
 
   render() {
+    const showLoadingSpinner = this.props.relay.hasMore() && !this.props.relay.isLoading()
```

Comment:
This seems incorrect. We should show a spinner when `isLoading` returns true, no?

Reply:
This just uncovered an unrelated issue where once a user scrolls to the bottom the list is repeatedly fetched without pause, like the update threshold doesn't change after new items  append. I remember running into this before when attempting to use the FlatList component with its built in end-detection.